src/CVRF/cvrf_priv.c

```diff
@@ -1632,8 +1632,7 @@ struct cvrf_reference *cvrf_reference_parse(xmlTextReaderPtr reader) {
 	__attribute__nonnull__(reader);
 
 	struct cvrf_reference *ref = cvrf_reference_new();
-	if (xmlTextReaderGetAttribute(reader, BAD_CAST "Type") != NULL)
-		ref->type = cvrf_item_parse_type_attribute(reader, CVRF_REFERENCE);
+	ref->type = cvrf_item_parse_type_attribute(reader, CVRF_REFERENCE);
```

Comment:
this seems to be changing the logic and has nothing to do with the commit message. or am I missing something?